pkg/noderesourcetopology/plugin.go

```diff
@@ -120,6 +121,10 @@ func New(args runtime.Object, handle framework.Handle) (framework.Plugin, error)
 		return nil, fmt.Errorf("want args to be of type NodeResourceTopologyMatchArgs, got %T", args)
 	}
 
+	if err := validation.ValidateNodeResourceTopologyMatchArgs(nil, tcfg); err != nil {
+		return nil, err
+	}
+
```

Comment:
@ffromani  I can also refactor `getScoringStrategyFunction` not to return `error` but I think it's not worth it cause this PR https://github.com/kubernetes-sigs/scheduler-plugins/pull/488 is changing the way how we want to handle handler initialization.

Reply:
Agreed, let's defer to a future PR.